src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -298,6 +297,13 @@ void prepare_primitive_fusing::fuse_bias(program &p) {
                 const_dep.get_output_layout().count() != static_cast<size_t>(out_features)) {
                 continue;
             }
+            auto parent_out_pshape = node->get_dependency(non_const_dep_idx).get_output_layout().get_partial_shape();
+            auto eltw_out_pshape = eltw_node.get_output_layout(0).get_partial_shape();
+            auto merged_shape = parent_out_pshape;
+            bool can_broadcast = ov::PartialShape::broadcast_merge_into(merged_shape, eltw_out_pshape, broadcast_type);
```

Comment:
Shouldn't this use parent's and constant's shapes? It seems the eltwise output might be already broadcasted/modified due to shape inference of eltwise(data, const)?

Reply:
Actually this is to prevent the case elt output (alrady broadcasted) is bigger than parent (not broadcasted).

Reply:
But I replaced this check with checking size, because actually when allow_new_shape_infer = false, the broadcast was not being done as expected.